templates/manx.html

```diff
@@ -22,89 +21,154 @@ <h2>Manx</h2>
-                                    <option value="" disabled selected>Select a technique</option>
-                                </select>
-                            </div>
-                        </td>
-                        <td>
-                            <div class="select is-small">
-                                <select id="procedure-filter" onchange="showProcedure()">
-                                    <option value="" disabled selected>Select a procedure</option>
-                                </select>
-                            </div>
-                        </td>
-                    </tr>
-                </table>
-            </center>
+        <div class="is-flex is-flex-direction-row is-justify-content-space-around">
+            <div class="select is-small">
+                <select id="session-id"
+                        x-model="selectedSessionID"
+                        x-on:change="getTactics()">
+                    <option value="" disabled selected>Select a session</option>
+                    <template x-for="s in sessions" :key="s.id">
+                        <option x-bind:value="s.id" x-bind:data-paw="s.info"
+                                x-bind:data-platform="s.platform"
+                                x-bind:data-executor="s.executors[0]"
+                                x-text="s.id + ' - ' + s.info"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select x-model="selectedTacticName" id="tactic-filter" x-on:change="getTechniques()">
+                    <option value="" disabled selected>Select a tactic</option>
+                    <template x-for="a in tactics" :key="a.ability_id">
+                        <option x-text="a.tactic"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select x-model="selectedTechniqueID" x-on:change="getProcedures()">
+                    <option value="" disabled selected>Select a technique</option>
+                    <template x-for="t in techniques" :key="t.technique_id">
+                        <option x-bind:value="t.technique_id"
+                                x-text="t.technique_id + ' | ' + t.technique_name"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select id="procedure-filter" x-model="selectedProcedureID" x-on:change="getProcedure();">
+                    <option value="" disabled selected>Select a procedure</option>
+                    <template x-for="p in procedures">
+                        <option x-text="p.name" x-bind:value="p.ability_id"></option>
+                    </template>
+                </select>
             </div>
-            <br>
-            <div id="xterminal"></div>
         </div>
+        <br>
```

Comment:
Can you add some padding between the select input row and the terminal? It'll help make it look less cluttered